### DISCUSSION TITLE
Documentation missing

### DISCUSSION THREAD

**Discussion by jcolomb** [knowledgefutures/pubpub#2510] [Support]
_Before filing your enhancement, please search existing issues to make sure a similar one doesn't already exist. If one does, please leave a comment or reaction in support of the issue rather than creating a new one._

_To fill out the request, please leave the below headers (**) in tact, and fill out each section. If a section isn't applicable, leave it blank or write n/a under it. Please do not delete any template sections._

**What's your enhancement request?**

The project needs better documentation, it takes for granted people will find what they want, but navigation is not easy. When looking for a feature, it is then not clear to the user if it is not yet implemented or if he cannot find it. It could be a FAQ, a wiki documentation a  video tour or a combination of these different medium, and it should be easily accessed from pubpub.org and the community page. Especially, I would like to know:

- ways to share edit rights with users, can you only add people who have an account?
- how to branch a project?
- is there a way to see the changes made by other team members, and a way to review them?
- is the mardown text still available, or is it transformed on the spot into the visible version ?

**What would this enhancement help you accomplish?**

- help getting used to the tool
- help understand what can and cannot be done, and therefore request features.

---

**Any attachments or reference images? (Drag them here)**

**What's the URL to your PubPub community?**
https://gintonic.pubpub.org
**Who requested this? (If not you)***

**Comment by metasj** [comment]
Hi @jcolomb, have you seen the [Help](https://help.pubpub.org/) community and [forum](https://github.com/pubpub/pubpub/discussions)?  To your first Q: You can get a URL to an editable version of any pub; but can only add community-wide rights for someone  once they make an account.

**Comment by gabestein** [comment]
Converting to a discussion, since we're using this for day-to-day code. But very much noted. We're planning to improve docs greatly this year.
